press/press/doctype/invoice/invoice.py

```diff
@@ -209,6 +209,30 @@ def finalize_invoice(self):
 			self.apply_partner_credits()
 			return
 
+		if self.payment_mode == "Card":
+			try:
+				self.create_stripe_invoice()
```

Comment:
Charging full invoice amount even though the customer has some wallet balance is not a standard behaviour. If this is required for some use-cases it needs to be configurable and not default for everyone.